app/src/main/java/io/apicurio/registry/config/RegistryConfigServiceImpl.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2021 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.registry.config;
+
+import static io.quarkus.scheduler.Scheduled.ConcurrentExecution.SKIP;
+
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.slf4j.Logger;
+
+import io.apicurio.registry.mt.TenantContext;
+import io.apicurio.registry.storage.RegistryStorage;
+import io.apicurio.registry.types.Current;
+import io.quarkus.scheduler.Scheduled;
+
+/**
+ * @author eric.wittmann@gmail.com
+ */
+@ApplicationScoped
+public class RegistryConfigServiceImpl implements RegistryConfigService {
+
+    private static final String DEFAULT_TENANT_ID = "___";
```

Comment:
why not using `TenantContext#DEFAULT_TENANT_ID`

Reply:
Good idea.  Will do it.